### PR TITLE
Add Io.GitHub.nokse22.minitext

### PR DESCRIPTION
**Pull request by Nokse22**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/Nokse22/mini-text
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Nokse22** [comment]
bot, build io.github.nokse22.minitext

**Comment by Nokse22** [comment]
bot, build io.github.nokse22.minitext

**Comment by Nokse22** [comment]
Everything is working as intended

File: io.github.nokse22.minitext.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    "tag"  : "v0.1.3",
+                    "commit" : "dda88edb2516a0ff34a2f095c6bed5023dc6035c",
+                    "url" : "https://github.com/Nokse22/mini-text"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (hfiguiere) [inline io.github.nokse22.minitext.json:42]:
```suggestion
    ]
```

**Pull request closed without merging**